### PR TITLE
Swahili (sw-KE) localization

### LINKED ISSUE #316: implement swahili localization
There is a detailed guide on how to go about adding a new language in the [localization documentation](https://github.com/goatshriek/stumpless/blob/latest/docs/localization.md).

If you'd like a concrete example of such a change, check out commit 5529e077eb75c6e79d5634e9d7fc708828b78f89, which shows how to update the various build and header files outlined in the documentation.

### PR DESCRIPTION
**Pull request by ghost**
Resolves #316

### CODE REVIEW

File: .github/workflows/locale.yml [hunk 1]
```diff
@@ -2,9 +2,9 @@ name: "Locale"
 
 on:
   push:
-    branches: [ latest ]
+    branches: [latest]
   pull_request:
-    branches: [ latest ]
+    branches: [latest]
```
Comment (goatshriek) [inline .github/workflows/locale.yml]: these should not be touched in this change since they are not relevant

File: README.md [hunk 2]
```diff
@@ -16,46 +16,48 @@
 [Basic Usage](#basic-usage) |
 [Contributing](#contributing)
 
-
 ## Key Features
+
```
Comment (goatshriek) [inline README.md]: remove this (and the other) newly-inserted newlines/whitespace

File: README.md [hunk 3]
```diff
@@ -16,46 +16,48 @@
...
 
+- easy logging to [lots of things](#what-can-it-log-to) like Splunk, rsyslog,
+  journald, the Windows Event Log, and more!
+- structured and unstructured logging to suit your needs
+- interoperable with common log daemons and libraries
+- builds for Linux, Windows, Mac, MinGW, MSYS2, Cygwin, and more
+- completely thread safe
+- can be adjusted or removed during compilation for zero runtime impact
+- localized for multiple languages :brazil: :bulgaria: :cn: :czech_republic:
+  :de: :es: :fr: :greece: :india: :it: :poland: :slovakia: :sweden: :us: :denmark: :israel: :kenya:
```
Comment (goatshriek) [inline README.md]: This line is the right place for the 🇰🇪 character, however the list bullet points seem to have been inadvertently changed. The only change that should be included in this file is addition of the flag - please revert the other changes.

File: include/private/config/locale/sw-ke.h [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+"urefu wa herufi umekuwa zaidi ya kikomo cha juu"
+
+#define L10N_STRING_LENGTH_ERROR_CODE_TYPE \
+"urefu wa herufi uliohusika"
+
+#define L10N_FORMAT_ERROR_MESSAGE(ARG) \
+"muundo wa" ARG "haufai"
+
+#define L10N_INVALID_TARGET_TYPE_ERROR_MESSAGE \
+"UJUMBE WA HITILAFU YA AINA YA LENGWAO BATILI"
```
Comment (goatshriek) [inline include/private/config/locale/sw-ke.h]: Error messages should be in all lower case

**Review by goatshriek — CHANGES_REQUESTED** [review]
Thank you for submitting this localization, it is mostly there! There are a few minor changes requested below. It seems like a markdown editor has made some judicious changes that weren't intended.

The check localization script has noted a few issues in your header file related to missing strings and alphabetization order. Fixing those should get the CI check to pass.

Note that you'll also need to add a line in the `include/private/config.h.in` file for the new locale as the others do - I don't see that in this PR presently. I suspect the CI check will fail after the l10n check script if this isn't corrected.

**Comment by mx700** [comment]
> Thank you for submitting this localization, it is mostly there! There are a few minor changes requested below. It seems like a markdown editor has made some judicious changes that weren't intended.
> 
> The check localization script has noted a few issues in your header file related to missing strings and alphabetization order. Fixing those should get the CI check to pass.
> 
> Note that you'll also need to add a line in the `include/private/config.h.in` file for the new locale as the others do - I don't see that in this PR presently. I suspect the CI check will fail after the l10n check script if this isn't corrected.

Thanks for reviewing! Let me know if there are any further issues or concerns as I hit visual studio code over the head with a hammer.

**Comment by mx700** [comment]
> Thank you for submitting this localization, it is mostly there! There are a few minor changes requested below. It seems like a markdown editor has made some judicious changes that weren't intended.
> 
> The check localization script has noted a few issues in your header file related to missing strings and alphabetization order. Fixing those should get the CI check to pass.
> 
> Note that you'll also need to add a line in the `include/private/config.h.in` file for the new locale as the others do - I don't see that in this PR presently. I suspect the CI check will fail after the l10n check script if this isn't corrected.

Give me a day.

**Comment by mx700** [comment]
> Thank you for submitting this localization, it is mostly there! There are a few minor changes requested below. It seems like a markdown editor has made some judicious changes that weren't intended.
> 
> The check localization script has noted a few issues in your header file related to missing strings and alphabetization order. Fixing those should get the CI check to pass.
> 
> Note that you'll also need to add a line in the `include/private/config.h.in` file for the new locale as the others do - I don't see that in this PR presently. I suspect the CI check will fail after the l10n check script if this isn't corrected.

Thanks for reviewing. Will add in missing strings and ordered alphabetization shortly.

**Comment by goatshriek** [comment]
Thanks very much for putting this together and getting changes made quickly!

**Comment by ghost** [comment]
> Thanks very much for putting this together and getting changes made quickly!

Happy to help.

**Pull request merged**